src/ensembl/src/content/app/custom-download/containers/content/attributes-accordion/state/attributesAccordionReducer.ts

```diff
@@ -90,20 +71,8 @@ function attributesAccordion(
   }
 }
 
-function GeneAttributes(
-  state: any = defaultAttributesAccordionState.attributes,
-  action: ActionType<RootAction>
-): CustomDownloadAttributes {
-  switch (action.type) {
-    case getType(attributesAccordionActions.setGeneAttributes):
-      return { ...state, gene: action.payload };
-    default:
-      return state;
-  }
-}
-
 function Orthologue(
```

Comment:
We do not begin function names with capital letters. Except when they are components (or classes).

Reply:
Updated it